src/components/cms/Dashboard/MyAnalytics.js

```diff
@@ -107,6 +109,17 @@ class MyAnalytics extends Component {
             </div>
           </Container>
         )}
+        {userSkills === 0 && !loading && (
```

Comment:
This block is redundant. Please check the condition before return and store the message in a variable. And if `message !== ''`, then display the message.